caffe2/python/data_parallel_model.py

```diff
@@ -823,8 +823,12 @@ def builder_fun(model):
 
     if device is None:
         device = scope.CurrentDeviceScope()
-
-    device_prefix = "gpu" if core.IsGPUDeviceType(device.device_type) else "cpu"
+    if device.device_type == caffe2_pb2.CUDA:
```

Comment:
Here should use core.IsGPUDeviceType instead of directly comparing to CUDA

Reply:
Had updated.